README.md

```diff
@@ -11,7 +11,7 @@ In config.js:
 ``` javascript
 System.config({
   map: {
-    "googlmaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!HeinrichFilter/systemjs-plugin-googlemaps/googlemaps"
+    "googlemaps": "https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places!systemjs-googlemaps"
```

Comment:
Thanks!

The `/googlemaps` at the end was annoying me but I couldn't quite figure out how to get rid of it. If you could help me to fix/load main, that would be great. (I'm quite new to SystemJS/JSPM)

Do you know where it is documented?